### ISSUE TITLE
Can we contact the author of those blueprints??

### ISSUE THREAD

**Issue by Zephyrchen2004** [gabriel-dehan/dyson-sphere-blueprints#15]
Some amazing blueprints was shared but part of them had some big or small mistake.
Can we contact the author or leave a message below the blueprint page?

**Comment by gabriel-dehan** [comment]
I am still torn about the idea of adding comments on the blueprint page as I said in https://github.com/gabriel-dehan/dyson-sphere-blueprints/issues/7 and somewhere on reddit. 
Might just create a private message system or a way to contact the authors🤔 Not sure what the best approach is.

**Comment by afilina** [comment]
@gabriel-dehan What about publishing authors' social media handles, like Twitter? Then there would be a way to contact them that doesn't require you to implement anything complicated or figuring out how to prevent abuse. There could be a very obvious "Contact the author" button near the description, that points to said handle.

**Comment by gabriel-dehan** [comment]
@afilina that might be an idea, but knowing that more than half of the users are chinese, with no legal access to twitter (they still can through a VPN but I guess most do not) and that the other half probably doesn't use Weibo (chinese twitter equivalent) nor wants to go through the trouble of creating an account just to contact an author, I'd rather just implement an in-site solution.

I thought about disqus but I don't want ads on the platform and the pricing is quite steep without the adds :)

**Comment by kris44dad** [comment]
As for myself, my ID here and my ID on Steam are the identical.  Easily found. 

As for the reasoning behind lack of comments, I can agree with the website admin. Since designing blueprints has a learning curve, I could see a comment section turning into one big bashing session for even the smallest flaw, regardless of any restrictions. The admin would be moderating all day.

**Comment by afilina** [comment]
Can we just have several buttons for people to vote? Then it could be displayed like this:
[Awesome work!] (56)
[Missing power poles] (15)
[Missing output filters] (3)

**Comment by Fullmetal85** [comment]
I agree with afilinas buttons idea. At least as a start it would be nice to let a creator know there is a problem with their blueprint. Ive had several where they uploaded part 3 as part 4 so its unusable as it but is other wise a great BP. It could just be a report problem with BP button that lets you choose from a few options like wrong BP uploaded, missing buildings, only works in one location and maybe a couple others and a final other problem that they can either then try to find it themselves or include a way to reach them in the details at their discretion.
